tests/unit/IonParserBinaryRawTest.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+define([
+    'intern',
+    'intern!object',
+    'intern/chai!assert',
+    'dist/amd/es6/IonTests',
+  ],
+  function(intern, registerSuite, assert, ion) {
+
+    let suite = {
+      name: 'Parser Binary Raw'
+    };
+
+    // Registers the provided test with the test suite.
+    let registerTest = function(testName, test, throwsException) {
+      suite[testName] = throwsException
+          ? function() { assert.throws(test, Error); }
+          : test;
+    }
+
+    let readUnsignedIntTest = function(bytes, expected, throwsException) {
```

Comment:
better to have two functions that call this one passing `true` or `false` for better test readability, e.g. `readUnsignedIntTest` and `readUnsignedIntTestReadMoreThanLength`

Reply:
I borrowed this from the other unit tests, but I like your approach much better. Done.